packages/dart_frog_auth/test/src/dart_frog_auth_test.dart

```diff
@@ -108,6 +108,22 @@ void main() {
         );
       },
     );
+
+    test("skips routes that doesn't match the custom predicate", () async {
```

Comment:
Should we have a test for  "applies to routes that match the custom predicate"?

Reply:
I don't think we need, since that is the default right? Wouldn't that be a redundant test?

Reply:
I think it wouldn't hurt. As long as there's some test checking this would be great. If in addition this tests are explicitly testing the functionality even better.

I currently think we are indeed lacking this test. Simply removing the following from [here](https://github.com/VeryGoodOpenSource/dart_frog/blob/f674f3e314b06f416373aa7e5475c3704c0b8f96/packages/dart_frog_auth/lib/src/dart_frog_auth.dart#L54) still manages to pass all the tests!!

```dart
    if (!await applies(context)) {
            return handler(context);
          }
```

Reply:
Interesting, fair point that the tests fails if we remove those lines, I am gonna check how I can make sure that is not the case.

Reply:
I actually had the test case there already, but it was wrong, I fixed it now makes sure that that code section is being correctly verified, good callout.